manim/mobject/svg/brace.py

```diff
@@ -249,37 +251,51 @@ def __init__(
         self.brace = Brace(obj, brace_direction, buff, **brace_config)
 
         if isinstance(text, (tuple, list)):
-            self.label = self.label_constructor(*text, font_size=font_size, **kwargs)
+            self.label: VMobject = self.label_constructor(
+                *text, font_size=font_size, **kwargs
+            )
         else:
             self.label = self.label_constructor(str(text), font_size=font_size)
 
         self.brace.put_at_tip(self.label)
         self.add(self.brace, self.label)
 
-    def creation_anim(self, label_anim=FadeIn, brace_anim=GrowFromCenter):
+    def creation_anim(
+        self,
+        label_anim: type[Animation] = FadeIn,
+        brace_anim: type[Animation] = GrowFromCenter,
+    ) -> AnimationGroup:
         return AnimationGroup(brace_anim(self.brace), label_anim(self.label))
 
-    def shift_brace(self, obj, **kwargs):
+    # TODO: This method is only called from the method change_brace_label, which is
+    # not called from any location in the current codebase.
+    # TODO: The Mobject could be more specific.
+    def shift_brace(self, obj: Mobject, **kwargs: Any) -> Self:
         if isinstance(obj, list):
             obj = self.get_group_class()(*obj)
         self.brace = Brace(obj, self.brace_direction, **kwargs)
         self.brace.put_at_tip(self.label)
         return self
 
-    def change_label(self, *text, **kwargs):
+    # TODO: This method is only called from the method change_brace_label, which is
+    # not called from any location in the current codebase.
```

Comment:
Although it's not called anywhere, it seems like an useful method to be exposed to the end user. In that case, the real problem would be that it's not documented at all. This could be a good follow-up PR.
```suggestion
```